reverse_replication/README.md

```diff
@@ -93,20 +93,38 @@ go run launcher.go -projectId=my-project -dataflowRegion=us-east1 -instanceId=my
 
 # Reverse Replication Limitations
 
-## Dataflow job of Spanner to Sink getting stuck in infinte loop
-The Dataflow job that reads the change streams and writes to PubSub gets stuck in infinite loop retrying the same set of records during certain scenarios.These scenario can arise when there are a lot of changestream records to be read in a short interval of time, which occurs in  the following situations:
-1. There is an unexpected spike on Spanner
-2. The pipeline is started with a date in past ( due to issues that required downtime such as bug fix )
-Currently, there is no way to revert this within the same job. More details and recovery steps below.
+## Throughput
```

Comment:
PubSub throughput is not the reason for the infinite loop isn't it? Even if we comment out writing to PubSub part - if there are huge set of records that amount to exceeding the BagState limit - there will still be this infinite loop issue.

Reply:
This limitation talks just about the max possible throughput we support under a stable state i.e. the Spanner throughput at which PubSub is at the limit. Infinite loop scenario is after this which happens under much higher throughputs.